[FEATURE] - Traductions manquantes NL et ES Pix App (PIX-14193)

pix-service-prod opened the pull request:
Exported the most recent locale files from Phrase

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr10430.review.pix.fr)
  * [App (.org)](https://app-pr10430.review.pix.org)
  * [Orga (.fr)](https://orga-pr10430.review.pix.fr)
  * [Orga (.org)](https://orga-pr10430.review.pix.org)
  * [Certif (.fr)](https://certif-pr10430.review.pix.fr)
  * [Certif (.org)](https://certif-pr10430.review.pix.org)
  * [Junior](https://junior-pr10430.review.pix.fr)
  * [Admin](https://admin-pr10430.review.pix.fr)
  * [API](https://api-pr10430.review.pix.fr/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr10430.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr10430/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr10430/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr10430/environment)

AndreiaPena commented:
Hello ! Possible de faire un test func coté Certif avant le merge ?

MathGuich commented:
@AndreiaPena J'ai utilisé Phrase pour ajouter nos traductions manquantes pour les futures mires. Pas le choix de tout embarquer (environ 120-140 clés). Ce serait intéressant de faire une revue, car impossible de savoir à qui les clés appartiennent (à part quelques Pix Companion que j'ai vu passer).

AndreiaPena commented on mon-pix/translations/es.json:
```diff
@@ -639,26 +661,35 @@
-            "5": "'<strong>'Movilizar cualquier otro medio que proporcione directamente la respuesta'</strong>', sin haber realizado el trabajo requerido por la instrucción.",
-            "6": "'<strong>'Use another browser'</strong>' other than the one you are using for your Pix Certification test.",
-            "7": "'<strong>'Impersonate'</strong>' another person."
+            "1": "'<strong>'Comunicarse con otra persona'</strong>', dentro o fuera de la sala, por medios físicos o electrónicos.",
```
bon comme je le craignais, c'est tout cassé par ici, je me charge de corriger ça sur Phrase directement

AndreiaPena replied:
@MathGuich j'ai mis à jour Phrase, peux-tu mettre à jour les fichiers s'il te plait ? :)

MathGuich replied:
C'est relancé

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

MathGuich reviewed (approved):
Ok pour Accès
